include/ceed-backend.h

```diff
@@ -200,4 +200,8 @@ CEED_EXTERN int CeedOperatorFieldGetLMode(CeedOperatorField opfield,
 CEED_EXTERN int CeedOperatorFieldGetVector(CeedOperatorField opfield,
     CeedVector *vec);
 
+CEED_INTERN int CeedMatrixMultiply(Ceed ceed, CeedScalar *matA,
+                                   CeedScalar *matB, CeedScalar *matC,
+                                   CeedInt m, CeedInt n, CeedInt kk);
+
```

Comment:
What memory space do its arguments reside in? That this is a reference implementation not intended for performance should probably be added to the docs.

Reply:
The caller is responsible for the memory of A, B, and C.

Reply:
Sure, but some of it is `const` and we should probably be explicit that they should be valid CPU pointers (we don't have a `CeedMemType`).

Reply:
Right. It was obvious when this was an interface only function. I'll fix it up.

Reply:
Added `const` and specified that the pointers must be on the CPU.